aspnetcore/tutorials/first-web-api.md

```diff
@@ -71,72 +69,72 @@ Install the [Entity Framework Core InMemory](https://docs.microsoft.com/en-us/ef
 
 Edit the *TodoApi.csproj* file. In Solution Explorer, right-click the project. Select **Edit TodoApi.csproj**. In the `ItemGroup` element, add the highlighted `PackageReference`:
 
-[!code-xml[Main](first-web-api/sample/src/TodoApi/TodoApi.csproj?highlight=17)]
+[!code-xml[Main](first-web-api/sample/TodoApi/TodoApi.csproj?highlight=1&range=14-15)]
 
 ### Add a model class
 
 A model is an object that represents the data in your application. In this case, the only model is a to-do item.
 
 Add a folder named "Models". In Solution Explorer, right-click the project. Select **Add** > **New Folder**. Name the folder *Models*.
 
-![Contextual menu](first-web-api/_static/add-folder.png)
+Note: You can put model classes anywhere in your project, but the *Models* folder is used by convention.
 
-> [!NOTE]
-> You can put model classes anywhere in your project, but the *Models* folder is used by convention.
-
-Add a `TodoItem` class. Right-click the *Models* folder and select **Add** > **Class**. Name the class `TodoItem` and tap **Add**.
+Add a `TodoItem` class. Right-click the *Models* folder and select **Add** > **Class**. Name the class `TodoItem` and select **Add**.
 
 Replace the generated code with:
 
-[!code-csharp[Main](first-web-api/sample/src/TodoApi/Models/TodoItem.cs)]
+[!code-csharp[Main](first-web-api/sample/TodoApi/Models/TodoItem.cs)]
+
+* The `[Key]` data-annotation denotes the property `Key` is a unique identifier.
```

Comment:
I've never seen data-annotation hyphenated, maybe replace here with just "attribute"  Also there's something missing in this sentence.